notebooks/least_core_basic.ipynb

```diff
@@ -0,0 +1,811 @@
+    "If you are reading this in the documentation, some boilerplate has been omitted for convenience.\n",
+    "</div>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "f6656599",
+   "metadata": {
+    "nbsphinx": "hidden"
+   },
+   "outputs": [],
+   "source": [
+    "%load_ext autoreload"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "08ee61fd",
+   "metadata": {
+    "nbsphinx": "hidden"
+   },
+   "outputs": [],
+   "source": [
+    "%autoreload\n",
+    "%matplotlib inline\n",
+    "\n",
+    "import logging\n",
+    "import os\n",
+    "import random\n",
+    "from typing import Dict\n",
+    "\n",
+    "import matplotlib.pyplot as plt\n",
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "from sklearn.datasets import make_classification\n",
+    "from sklearn.linear_model import LogisticRegression\n",
+    "from sklearn.metrics import mean_squared_error\n",
+    "from sklearn.model_selection import train_test_split\n",
+    "\n",
+    "plt.ioff()  # Prevent jupyter from automatically plotting\n",
+    "plt.rcParams[\"figure.figsize\"] = (20, 8)\n",
+    "plt.rcParams[\"font.size\"] = 12\n",
+    "plt.rcParams[\"xtick.labelsize\"] = 12\n",
+    "plt.rcParams[\"ytick.labelsize\"] = 10\n",
+    "\n",
+    "mean_colors = [\"dodgerblue\", \"indianred\", \"limegreen\", \"darkorange\", \"darkorchid\"]\n",
+    "shade_colors = [\"lightskyblue\", \"firebrick\", \"seagreen\", \"gold\", \"plum\"]\n",
+    "\n",
+    "random_state = 16\n",
+    "random.seed(random_state)\n",
+    "np.random.seed(random_state)\n",
+    "\n",
+    "is_CI = os.environ.get(\"CI\")\n",
+    "\n",
+    "if is_CI:\n",
+    "    enable_cache = False\n",
+    "    train_size = 10\n",
+    "else:\n",
+    "    enable_cache = True\n",
+    "    train_size = 200"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ea11d1ff",
+   "metadata": {},
+   "source": [
+    "We will be using the following functions and classes from pyDVL."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "3155c98e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%autoreload\n",
+    "from pydvl.utils import (\n",
+    "    Dataset,\n",
+    "    Utility,\n",
+    ")\n",
+    "from pydvl.value.least_core import exact_least_core, montecarlo_least_core\n",
+    "from pydvl.reporting.plots import shaded_mean_std\n",
+    "from pydvl.reporting.scores import compute_removal_score, compute_random_removal_score"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1e7d81aa",
+   "metadata": {},
+   "source": [
+    "## Dataset\n",
+    "\n",
+    "We will use a synthetic dataset. We sample 200 data points from a 50-dimensional Gaussian distribution.\n",
+    "\n",
+    "The dataset is generated using the [`make_classification`](https://scikit-learn.org/stable/modules/generated/sklearn.datasets.make_classification.html) function from scikit-learn."
+   ]
```

Comment:
I think it is helpful to add a small description of the dataset here. Just a few words to understand how to increase/decrease complexity if user wants to experiment

Reply:
I added some more information to that description.